eng/imports/HostAgnostic.props

```diff
@@ -80,51 +79,4 @@
 
     <InternalsVisibleTo Include="DynamicProxyGenAssembly2" Key="$(MoqPublicKey)" />
   </ItemGroup>
-
- <!--
-    Some of the VSSDK packages may pull in Microsoft.VisualStudio.SDK.EmbedInteropTypes which essentially
-    does the below but for all of the listed references. We can't just use what they have since they include
-    things like EnvDTE which is reasonable for consumers, but strange since we actually _implement_ DTE and
-    use it as an exchange type with generics in a few places. Instead, we run directly after the target declared
-    in the EmbedInteropTypes package and ensure only the dependencies we need are embedded and the others are
-    skipped.
-  -->
-  <Target Name="CustomLinkVSSDKEmbeddableAssemblies" AfterTargets="LinkVSSDKEmbeddableAssemblies">
```

Comment:
Are we sure this is no longer needed? While we may no longer need Microsoft.VisualStudio.SDK.EmbedInteropTypes, I would expect we still need to be careful about what we do or do not embed.

Reply:
I suppose the definitive answer would be to build with and without this change and see if there are changes in what gets embedded.

Though if the guidance for 64-bit VS is to *not* embed interop types and that is the default behavior then we're probably best off removing all of this non-default embedded behavior and just dealing with the fallout (if any).